codeHF/dpl-config_run3.json

```diff
@@ -951,7 +940,7 @@
         "selectionFlagLcbar": "0",
         "cutEtaCandMax": "-1"
     },
-    "hf-task-correlation-d0": {
+    "task-correlation-d0": {
```

Comment:
The device name should start with HF, so `hf-task-correlation-d0` was correct and the O2Physics code should be modified accordingly.